playwright/utils/test_setup.ts

```diff
@@ -0,0 +1,32 @@
+import { chromium, ChromiumBrowser, Page, BrowserContext } from 'playwright'
+export let browser: ChromiumBrowser
+export let context: BrowserContext
+export let page: Page
+
+export const baseURL = process.env.BASE_URL || 'http://localhost:8080'
+
+// PWDEBUG is playwright specific flag
+// sets up window.playwright and other stuff
+const isDebug = !!process.env.PWDEBUG
+let slowMo: number | undefined
+
+// timeout to have time to account for async nature of browser actions
+jest.setTimeout(15000)
+if (isDebug) {
+  // slow down playright actions
+  slowMo = 2000
+} else {
+  // close after only when not debugging
+  afterAll(() => browser.close())
+  afterEach(() => page.close())
+}
+
+beforeAll(async () => {
+  // launch chrome,headless by default
+  browser = await chromium.launch({ slowMo })
```

Comment:
we have to hardcode something
can of course do `BROWSER=chromium|firefox|webkit` and later pick it

Reply:
Also can think about an optional `executablePath` argument, to use Chromium but use your local installed Chrome browser. Should in theory be the same I suppose (Chromium <-> Chrome) but there can be nuances in practice when testing.

Reply:
Nah, can't count on Chrome extensions not screwing everything up
Because if you just use local Chrome, and have MMask installed, it will conflict with `window.ethereum` injected by playwright
And if you use local Cjrome with `--sandbox` or whatever no-extension flag, then it's almost the same as Chromium

Reply:
I guess my thinking is:
- Even having 'expected' extensions like MM, might simulate/catch some side effects/bugs as opposed to run it in a clean env. like Chromium.
- Same for running Chromium vs Chrome, like you say they are almost the same but I'd regard it as a more realistic test if you know what I mean.

Reply:
Now this is interesting indeed. Can do
```sh
EXECUTABLE_PATH=$(which google-chrome) yarn test:playwright
```